packages/fxa-payments-server/Dockerfile

```diff
@@ -9,19 +9,45 @@ RUN npm ci
 COPY ["fxa-payments-server/.storybook", ".storybook/"]
 COPY ["fxa-payments-server/public", "public/"]
 COPY ["fxa-payments-server/src", "src/"]
+COPY ["fxa-shared", "../fxa-shared/"]
+WORKDIR /fxa-shared
+USER root
+RUN npm ci
+
+USER app
 COPY ["fxa-content-server", "../fxa-content-server/"]
 WORKDIR /fxa-content-server
 RUN npm ci
 WORKDIR /app
 ENV PUBLIC_URL /
 ENV INLINE_RUNTIME_CHUNK false
 RUN npm run build
-
-FROM node:12-stretch-slim
+FROM node:12-stretch
```

Comment:
I'm not sure if this is OK, or if we should stick with node:12-stretch-slim and manually install git.

Reply:
This is a question for @jrgm and @jbuck.

Reply:
the difference is node:12-stretch is a 1GB image, -slim  is 1/6th that size. Mattered for time to provision a new ec2 instance for scaling in our AWS config, but may not matter in kubernetes on gcp.
also, the general principle of only installing what is required.